README.md

```diff
@@ -13,11 +13,40 @@ how tracing services looks.
 
 `npm install zipkin --save`
 
+## Basic Setup:
+
+```javascript
+const {
+  Tracer,
+  BatchRecorder,
+  jsonEncoder: {JSON_V2}
+} = require('zipkin');
+const CLSContext = require('zipkin-context-cls');
+const {HttpLogger} = require('zipkin-transport-http');
+
+// Setup the tracer to use http and implicit trace context
+const tracer = new Tracer({
+  ctxImpl: new CLSContext('zipkin'),
+  recorder: new BatchRecorder({
+    logger: new HttpLogger({
+      endpoint: 'http://localhost:9411/api/v2/spans',
+      jsonEncoder: JSON_V2
+    })
+  }),
+  localServiceName: 'service-a' // name of this application
+});
+
+// now use tracer to construct instrumentation! For example, fetch
+const wrapFetch = require('zipkin-instrumentation-fetch');
+
+const remoteServiceName = 'youtube';
+const zipkinFetch = wrapFetch(fetch, {tracer, remoteServiceName});
```

Comment:
@yonran @pbadenski hoping we can somehow deprecate passing the `serviceName` argument into each instrumentation. People tend to get it wrong which results in traces with mixed service names for the same service, or broken service dependency graphs. By moving this to the tracer, hopefully it will be more obvious.

Do you know how to deprecate something in javascript?

Also, I'd like to make sure we don't encourage passing `serviceName` in typescript. Is the best way to just not define the arg?

Reply:
That’s fine that you took it out of the TypeScript declaration.